File: IOModels/CustomerDetailRequest.cs
namespace SmartAlerts.API.IOModels
{
    public class CustomerDetailRequest
    {

        public long UserID { get; set; }

        public DateOnly CustomerDOB { get; set; }

        public DateOnly? CustomerAnniversary { get; set; }

        public string? ProfilePicURL { get; set; }
        public string? DeviceFCMToken { get; set; }
    }
}

File: main/java/com/smart/alerts/models/CustomerDetailRequest.java
package com.smart.alerts.models;

public class CustomerDetailRequest {

    public long UserID;
    public String CustomerDOB;
    public String CustomerAnniversary;
    public String ProfilePicURL;
    public String DeviceFCMToken;

    public CustomerDetailRequest() {
        UserID = 0;
        CustomerDOB = null;
        CustomerAnniversary = null;
        ProfilePicURL = null;
        DeviceFCMToken=null;
    }
}
